reflex/components/forms/editable.py

```diff
@@ -36,6 +38,31 @@ class Editable(ChakraComponent):
     # The initial value of the Editable in both edit and preview mode.
     default_value: Var[str]
 
+    # Time in milliseconds to wait between end of input and triggering on_change
+    debounce_timeout: Var[int]
```

Comment:
```suggestion
```

should avoid defining this as an actual prop on `Editable`, since the `Editable` component doesn't recognize it.

Reply:
Thanks for making this suggestion again. I just looked at how DebounceInput is a wrapper of an equivalent react component, I get what you're saying about real props. Now this is what I think. As a non-react developer, I'm thinking about the behavior I want with my component rather than if a certain arg maps to a legit prop in the react component's or not. Personally I wouldn't go out of my way to find that out if I'm using reflex. I don't feel it needs to be a prop here. What I want to do is since we are not 100% confident on the default 50ms, there should be the option to give users. Of course if we just add a non-prop argument in the `create` method, it can create debounced wrappers, but do we expose docs on that, or we pretty much just tell users about the props.

Reply:
> Of course if we just add a non-prop argument in the create method, it can create debounced wrappers, but do we expose docs on that, or we pretty much just tell users about the props.

There are many components in reflex that special case kwargs to the `create` function that are not actual props, so there's precedent for this type of structure.

See some examples in
* reflex/components/datadisplay/code.py
* reflex/components/datadisplay/table.py
* reflex/components/disclosure/tabs.py
* reflex/components/forms/slider.py

I think how we document it is important. Currently those other components with special handling i think just provide examples of use, without fully documenting the special cases, which is problematic.

At least on the `Input`, `TextArea` and `Editable` docs, we can mention the `debounce_timeout` parameter and link to the `DebounceInput` docs for a full explanation.

Reply:
That sounds good!

Reply:
removed here https://github.com/reflex-dev/reflex/pull/1650/commits/18ff4992ef065c32a657b50bb01c141c88010ad4, l'm also working on updating the docs/tutorial, will add it there.